### ISSUE TITLE
fix examples in closed.testing

### ISSUE THREAD

**Issue by jangorecki** [kkholst/lava#1]
There is a call `loadNamespace(mets)` in examples - https://github.com/kkholst/lava/blob/master/man/closed.testing.Rd#L39
As mets is suggested dependency, it should not be required to run examples according to R-exts. This can be conditionally escaped with requireNamespace.
It is possible there are other cases of that issue in other examples, this one was to first that fails.

```
* checking examples ... ERROR
Running examples in ‘lava-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: closed.testing
> ### Title: Closed testing procedure
> ### Aliases: closed.testing p.correct
> 
> ### ** Examples
> 
> m <- lvm()
> regression(m, c(y1,y2,y3,y4,y5,y6,y7)~x) <- c(0,0.25,0,0.25,0.25,0,0)
> regression(m, to=endogenous(m), from="u") <- 1
> variance(m,endogenous(m)) <- 1
> set.seed(2)
> d <- sim(m,200)
> l1 <- lm(y1~x,d)
> l2 <- lm(y2~x,d)
> l3 <- lm(y3~x,d)
> l4 <- lm(y4~x,d)
> l5 <- lm(y5~x,d)
> l6 <- lm(y6~x,d)
> l7 <- lm(y7~x,d)
> 
> (a <- merge(l1,l2,l3,l4,l5,l6,l7,subset=2))
    Estimate Std.Err     2.5% 97.5%  P-value
x    -0.0220  0.0993 -0.21668 0.173 8.25e-01
x.1   0.3723  0.1157  0.14564 0.599 1.29e-03
x.2   0.1198  0.1110 -0.09780 0.337 2.81e-01
x.3   0.4223  0.0926  0.24076 0.604 5.14e-06
x.4   0.2934  0.1214  0.05558 0.531 1.56e-02
x.5   0.2057  0.1062 -0.00246 0.414 5.28e-02
x.6   0.0524  0.1182 -0.17922 0.284 6.57e-01
> p.correct(a)
Error in loadNamespace(name) : there is no package called ‘mets’
Calls: p.correct ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
```

**Comment by kkholst** [comment]
Thanks, fixed now in [14a696f](https://github.com/kkholst/lava/commit/14a696f381a10dffb8d3c4520be2623b34525c58).

**Comment by jangorecki** [comment]
@kkholst any chance for a list of packages that are mandatory to run package check? I see the (impressive) list in travis yaml has all possible dependencies, but I would like to install as few as necessary to run R CMD check.
